spec/passaporte_web/identity_spec.rb

```diff
@@ -205,12 +205,12 @@
       identity.email_list.count.should == 2
       # Primary e-mail
       identity.email_list[0]['address'].should == 'jaime.lannister@mailinator.com'
-      identity.email_list[0]['is_primary'].should be_true
-      identity.email_list[0]['is_active'].should be_true
+      expect(identity.email_list[0]['is_primary']).to be_truthy
+      expect(identity.email_list[0]['is_active']).to be_truthy
       # Secondary email
       identity.email_list[1]['address'].should == 'kingslayer@mailinator.com'
-      identity.email_list[1]['is_primary'].should be_false
-      identity.email_list[1]['is_active'].should be_true
+      expect(identity.email_list[1]['is_primary']).to be_falsy
+      expect(identity.email_list[1]['is_active']).to be_truthy
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.